roles/import_image_and_run_gcp_instance/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+        msg: "A GCP compute instance with name {{ import_image_and_run_gcp_instance_instance_name }} already exists"
+      when: import_image_and_run_gcp_instance__gcp_instance_info.resources | length == 1
+
+    - name: Create temporary directory where to store the archive
+      ansible.builtin.tempfile:
+        state: directory
+        suffix: .storage
+      register: import_image_and_run_gcp_instance__dir
+      notify:
+        - "Delete temporary directory"
+
+    - name: Check if 'import_image_and_run_gcp_instance_image_path' exists
+      ansible.builtin.stat:
+        path: "{{ import_image_and_run_gcp_instance_image_path }}"
+      register: import_image_and_run_gcp_instance__image_path
+
+    - name: Fail when 'import_image_and_run_gcp_instance_image_path' does not exist
+      ansible.builtin.fail:
+        msg: "The {{ import_image_and_run_gcp_instance_image_path }} does not exist"
+      when: not import_image_and_run_gcp_instance__image_path.stat.exists
+
+    - name: Rename .raw file to disk.raw
+      ansible.builtin.copy:
+        src: "{{ import_image_and_run_gcp_instance_image_path }}"
+        dest: "{{ import_image_and_run_gcp_instance__dir.path }}/disk.raw"
+
+    - name: Set 'import_image_and_run_aws_instance__s3_object_key' variable
+      ansible.builtin.set_fact:
+        import_image_and_run_gcp_instance__archive_path: "{{ import_image_and_run_gcp_instance__dir.path }}/disk.tar.gz"
+        import_image_and_run_gcp_instance__object_key: "{{ import_image_and_run_gcp_instance_bucket_name }}/disk.tar.gz"
+
+    - name: Gather facts on host
+      setup:
+
+    - name: Create a tar.gz archive of a the .raw image
+      shell: "{{ 'gtar' if 'Darwin' in ansible_facts['system'] else 'tar' }} --format=oldgnu -Sczf disk.tar.gz disk.raw"
+      args:
+        chdir: "{{ import_image_and_run_gcp_instance__dir.path }}"
+
+    - name: Upload archive to Storage bucket
+      google.cloud.gcp_storage_object:
+        bucket: "{{ import_image_and_run_gcp_instance_bucket_name }}"
+        dest: "{{ import_image_and_run_gcp_instance__object_key }}"
+        src: "{{ import_image_and_run_gcp_instance__archive_path }}"
+        action: upload
+      register: import_image_and_run_gcp_instance__upload_image
+
+    - name: Import image
+      google.cloud.gcp_compute_image:
+        state: present
+        name: "{{ import_image_and_run_gcp_instance_import_image_name }}"
+        raw_disk:
+          source: "{{ import_image_and_run_gcp_instance__upload_image }}"
+      register: import_image_and_run_gcp_instance__import_result
+
+    - name: Check status of the import image task
+      google.cloud.gcp_compute_image_info:
+        filters:
+          - name = "{{ import_image_and_run_gcp_instance_import_image_name }}"
+      register: import_image_and_run_gcp_instance__import_image_info
+
+    - name: Start GCP compute instance
+      google.cloud.gcp_compute_instance:
+        zone: "{{ import_image_and_run_gcp_instance_zone)}}"
```

Comment:
```suggestion
        zone: "{{ import_image_and_run_gcp_zone)}}"
```
Same here

Reply:
It should be called import_image_and_run_gcp_instance_zone (_instance got cut during various copy/paste steps), it should be fixed now everywhere. Thanks.